astrocut/tests/test_cutout_processing.py

```diff
@@ -254,11 +256,79 @@ def test_center_on_path(tmpdir):
     assert hdu[0].header["DATE"] == Time.now().to_value('iso', subfmt='date')
     assert hdu[0].header["OBJECT"] == ""
     hdu.close()
-    
 
-    
 
-    
+def test_default_combine():
```

Comment:
Some more description for this test (eg, that build_default_combine_function is creating masks of the input array) can help others understand why some of the assertions work the way they do.